frida_mode/README.md

```diff
@@ -10,23 +10,23 @@ a small harness around their target code of interest, FRIDA mode instead takes a
 different approach to avoid these limitations.
 
 # Current Progress
-As FRIDA mode is new, it is missing a lot of features. Most importantly,
-persistent mode. The design is such that it should be possible to add these
-features in a similar manner to QEMU mode and perhaps leverage some of its
-design and implementation.
-
-  | Feature/Instrumentation  | frida-mode |
-  | -------------------------|:----------:|
-  | NeverZero                |            |
-  | Persistent Mode          |            |
-  | LAF-Intel / CompCov      |            |
-  | CmpLog                   |            |
-  | Selective Instrumentation|     x      |
-  | Non-Colliding Coverage   |            |
-  | Ngram prev_loc Coverage  |            |
-  | Context Coverage         |            |
-  | Auto Dictionary          |            |
-  | Snapshot LKM Support     |            |
+As FRIDA mode is new, it is missing a lot of features. The design is such that it
+should be possible to add these features in a similar manner to QEMU mode and
+perhaps leverage some of its design and implementation.
+
+  | Feature/Instrumentation  | frida-mode | Notes                                   |
+  | -------------------------|:----------:|:---------------------------------------:|
+  | NeverZero                |     x      |                                         |
+  | Persistent Mode          |     x      | (x64 only)(Only on function boundaries) |  
+  | LAF-Intel / CompCov      |     -      | (Superseded by CmpLog)                  |  
```

Comment:
compcov is very different to implement, much more work and less often effective than cmplog.
so putting compcov on the todo list but putting it somewhere on the bottom is the right thing IMHO :)

Reply:
Happy to add it somewhere down the line if there's a need. When I was discussing with @vanhauser-thc, he seemed to say that CompLog was much better? If there are use-cases where CompCov is better, or CompLog unsuitable, then I am happy to re-evaluate? CompLog support shouldn't be too hard, but it will take some time and would need implementing for each architecture.

Reply:
if you have decoded the instruction already, it should be straight forward to emit compcov coverage at that spot in frida.
I agree complog is better, though. Ignore it for now :)